administration/src/bp-modules/components/CustomDatePicker.tsx

```diff
@@ -1,51 +1,69 @@
+import { SystemStyleObject } from '@mui/system'
 import { DesktopDatePicker } from '@mui/x-date-pickers'
 import { deDE } from '@mui/x-date-pickers/locales'
-import React, { ReactElement } from 'react'
+import React, { CSSProperties, ReactElement } from 'react'
 
 import useWindowDimensions from '../../hooks/useWindowDimensions'
 
 type CustomDatePickerProps = {
-  date: Date | null
-  onBlur: () => void
-  onChange: (date: Date | null) => void
-  onClear: () => void
+  date?: Date
+  disabled?: boolean
+  label?: string
+  onBlur?: () => void
+  onChange?: (date: Date | null) => void
+  onClear?: () => void
   isValid: boolean
   minDate?: Date
   maxDate?: Date
   disableFuture: boolean
+  textFieldHelperText?: string
+  textFieldStyle?: CSSProperties
+  textFieldThemeOverride?: SystemStyleObject
 }
 
 const CustomDatePicker = ({
   date,
+  disabled,
+  label,
   onBlur,
   onChange,
   onClear,
   isValid,
   minDate,
   maxDate,
   disableFuture,
+  textFieldHelperText,
+  textFieldStyle,
+  textFieldThemeOverride,
 }: CustomDatePickerProps): ReactElement => {
   const { viewportSmall } = useWindowDimensions()
-  const formStyle = viewportSmall ? { fontSize: 16, padding: '9px 10px' } : { fontSize: 14, padding: '6px 10px' }
-  const textFieldBoxShadow =
-    '0 0 0 0 rgba(205, 66, 70, 0), 0 0 0 0 rgba(205, 66, 70, 0), inset 0 0 0 1px #cd4246, inset 0 0 0 1px rgba(17, 20, 24, 0.2), inset 0 1px 1px rgba(17, 20, 24, 0.3)'
+  const textFieldBoxShadow = `
+    0 0 0 0 rgba(205, 66, 70, 0),
+    inset 0 0 0 1px rgba(17, 20, 24, 0.2),
+  `
+
   return (
     <DesktopDatePicker
+      disabled={disabled}
+      label={label}
       views={['year', 'month', 'day']}
       value={date}
```

Comment:
I think it would also be nice to add the prop `openTo="year"` as selecting a year is probably necessary for every of our date fields.